lib/features/auth/view/widget/auth_input_field.dart

```diff
@@ -0,0 +1,108 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:telware_cross_platform/core/theme/palette.dart';
+
+class AuthInputField extends ConsumerStatefulWidget {
+  final String name;
```

Comment:
As long as you didn't use "ref" inside the widget, then, make it a normal StatefulWidget